moveit_core/package.xml

```diff
@@ -46,16 +46,20 @@
   <depend>tf2_geometry_msgs</depend>
   <depend>trajectory_msgs</depend>
   <depend>visualization_msgs</depend>
-  <depend>xmlrpcpp</depend>
+  <depend>rcutils</depend>
+  <!-- Not yet ported -->
+  <!-- <depend>xmlrpcpp</depend> -->
 
   <test_depend>angles</test_depend>
   <test_depend>moveit_resources</test_depend>
   <test_depend>tf2_kdl</test_depend>
   <test_depend>orocos_kdl</test_depend>
   <test_depend>rosunit</test_depend>
   <test_depend>code_coverage</test_depend>
+  <test_depend>ament_cmake_gtest</test_depend>
 
   <export>
     <moveit_core plugin="${prefix}/collision_detector_fcl_description.xml"/>
+    <build_type>ament_cmake</build_type>
   </export>
 </package>
```

Comment:
the following should be deleted: moveit_ros/planning_interface/COLCON_IGNORE

Reply:
the COLCON_IGNORE files enable us to run the docker. Without them rosdep install fails

Reply:
this should be in a separate PR then, its out of scope of this PR.

Reply:
Sure thing